src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Figure {
+    String drawFigure();
+
+    double getSquare();
+
+    Color getColor();
```

Comment:
move this method to the abstract classs and create there a field color with setter and getter